x-pack/elastic-agent/_meta/config/common.p2.yml.tmpl

```diff
@@ -5,8 +5,11 @@ outputs:
   default:
     type: elasticsearch
     hosts: [127.0.0.1:9200]
-    username: elastic
-    password: changeme
+    api_key: "example-key"
```

Comment:
I'm a bit confused if this attribute should be `api_key` or `service_token`.
the libbeat output config expects `api_key`, but some config from within the agent refers to it as `service_token`.

Reply:
api_key is for the output, service_token is for what is passed to fleet-server. This is not the same thing, see other comments.